roadmap/implementers-guide/src/node/collators/collator-protocol.md

```diff
@@ -98,7 +98,11 @@ digraph G {
 }
 ```
 
-When peers connect to us, they can `Declare` that they represent a collator with given public key and intend to collate on a specific para ID. Once they've declared that, and we checked their signature, they can begin to send advertisements of collations. The peers should not send us any advertisements for collations that are on a relay-parent outside of our view or for a para outside of the one they've declared.
+When peers attempt to connect to a validator node, they are required to present a PrePoV to the validator node as part of the handshake of the network connection within substrate. This will provide all information regarding that collator's identity, i.e. their public key (`CollatorId`), the specific para ID they intend to collate as well as a witness to the PrePVF's `validate` function. Upon successful verification the collator may begin to send advertisements of collations. The validator maintains historic state for every `CollatorId` they have come across in a persistent data structure which allows keep track of the total and average occurence (per unit time) of undesired behavior for this collator. Examples of undesired behavior include but are not limited to submitting advertisements for collations that are on a relay-parent outside of our view or for a different para than the one they have declared in the handshake of their network connection.
```

Comment:
1. `PrePov` is not necessarily part of the handshake although I suspect it should be at least an optional element so we can support restrictive collator selection algorithms that are entirely determined by the validation Wasm
2. The guide should not refer to Substrate directly; our aim is to keep it high-level and agnostic. Potentially as a reference document to be used by other client teams which are not using Substrate.